web/js/palettes/model.js

```diff
@@ -346,7 +347,7 @@ export function palettesModel(models, config) {
 
     loEach(state.l, function (layerDef) {
       var layerId = layerDef.id;
-      var minValue, maxValue;
+      var values, minValue, maxValue;
```

Comment:
Why are you defining `values` outside of the scope it was originally used in? It looks like the line `values = util.toArray(attr.value.split(';'));` should have the `var` keyword added to it to keep the variable in the same scope. Moving it out of that scope could change the behavior.

Reply:
Otherwise `values` is undefined. Seems to make sense to define it here.

Reply:
Why do you need `values` to be defined outside of the scope it was originally defined in? This `var` declaration belongs inside of the `loEach` loop, on line 358 where it's being given a value for the first time, unless you're using it before that.